.vib/postgrest/goss/postgrest.yaml

```diff
@@ -0,0 +1,9 @@
+command:
+  check-app-version:
+    exec: postgrest --help
+    exit-status: 0
+    stdout:
+      - {{ .Env.APP_VERSION }}
```

Comment:
You should use the `check-app-version` common template, as it also allows for specifying the command flag.

Reply:
Oops, yes, you are right! I missed it completely when implementing this.